pkg/server/commands/connected_objects.go

```diff
@@ -101,6 +101,42 @@ type ConnectedObjectsCommand struct {
 	Limit uint32
 }
 
+type ConnectedObjectsQueryOption func(d *ConnectedObjectsCommand)
+
+func WithCOResolveNodeLimit(limit uint32) ConnectedObjectsQueryOption {
```

Comment:
Probably worth putting the `ConnectedObjects` related code in it's own package, and then these options become uniquely scoped to that package.

Reply:
I was going to send another PR that renames `ConnectedObjectsCommand` to `ConnectedObjectsQuery`. Can I include your suggestion in that PR?

Reply:
Yeah. Please rename these to simply `WithResolveNodeLimit` instead of `WithCOResolveNodeLImit` in that PR when you repackage the ConnectedObjectsQuery.